relion/EM_transform.py

```diff
@@ -24,6 +24,7 @@ def group_by_class_number(self):
                     except ValueError:
                         pass
                 final_dict[x[0]] = dict
+        print(final_dict)
```

Comment:
I note this file contains two added `print()` calls - probably leftovers from a debug session?